Bootcamp2015.AmazingRace.Base/Helpers/LocationHelper.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Bootcamp2015.AmazingRace.Base.Models;
+using Bootcamp2015.AmazingRace.Base.ServiceInterfaces;
+using Microsoft.WindowsAzure.MobileServices;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using Windows.Devices.Geolocation;
+using Windows.Security.Credentials;
+
+namespace Bootcamp2015.AmazingRace.Base.Helpers
+{
+    public static class LocationHelper
+    {
+        private static Geolocator _locator;
+
+        private static PasswordCredential credentials;
+        private static MobileServiceClient client;
+
+        //private static Team team = _settingsService.GetDeserializedValueOrDefault<Team>("TEAM");
+
+        public static void Start()
+        {
+            if (credentials == null)
+            {
+                credentials = PasswordVaultHelper.GetPasswordCredential();
+
+                if (credentials != null)
+                {
+                    var user = PasswordVaultHelper.GetUser();
+                    client = new MobileServiceClient(Connections.MobileServicesUri, Connections.MobileServicesAppKey);
+                    client.CurrentUser = user;
+
+                    if (_locator == null)
+                    {
+                        _locator = new Geolocator() { ReportInterval = 1000 };
+                        //_locator.getGeopositionAsync();
+                        _locator.PositionChanged += OnLocatorPositionChanged;
```

Comment:
You might get an issue with a memory leak here because we never unhook from this event and we create a new one every time this method is called.